docs/providers/aws/events/apigateway.md

```diff
@@ -191,6 +191,41 @@ functions:
             identityValidationExpression: someRegex
 ```
 
+You can also configure an existing Cognito User Pool as the authorizer, as shown
+in the following example:
+
+```yml
+functions:
+  create:
+    handler: posts.create
+    events:
+      - http:
+          path: posts/create
+          method: post
+          authorizer:
+            arn: arn:aws:cognito-idp:us-east-1:xxx:userpool/us-east-1_ZZZ
```

Comment:
So my understanding of federated identities is that they just when you set them up the browser gets given a temp access key and secret. So basically it's equivalent to IAM access. So you don't actually need the ARN at all. I think we would do something like

``` yaml
authorizer:
  type: IAM
```

and then just set the authorizer type to AWS_IAM